airgun/entities/flatpak.py

```diff
@@ -83,6 +83,52 @@ def create(self, values):
         view = FlatpakRemoteDetailsView(self.browser)
         view.wait_displayed(delay=3)
 
+    def create_redhat_remote(self, values):
+        """Create a Red Hat Flatpak remote using the info alert action.
+
+        This method clicks the 'Add Red Hat flatpak remote' button from the
+        info alert instead of manually filling the URL.
+
+        :param dict values: values to create the remote with
+        """
+        view = self.navigate_to(self, 'All')
+        view.wait_displayed()
+        view.create_new_btn.click()
+        create_modal = CreateFlatpakRemoteModal(self.browser)
+        create_modal.wait_displayed()
+        create_modal.fill(values)
+
+        if create_modal.info_alert.is_displayed:
+            create_modal.add_rh_fr.click()
+        else:
+            raise ValueError('Red Hat Flatpak remote info alert is not displayed')
+
+        create_modal.create_btn.click()
+        view = FlatpakRemoteDetailsView(self.browser)
```

Comment:
Same here, I'd suggest a 'Details' Navigator, which could replace both the flat navigation to the `FlatpakRemoteDetailsView` and the version of it that needs to look at a specific entity, like in the above `create` method. If you want inspiration, you can look at this method in host_new: https://github.com/SatelliteQE/airgun/blob/f2a5e47274ed1c66ee7bd6148ae07e85adc86667/airgun/entities/host_new.py#L1117